packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -248,9 +249,14 @@ export default function useInitEditedEntityFromURL() {
 		useResolveEditedEntityAndContext( params );
 
 	const { setEditedEntity } = useDispatch( editSiteStore );
+	const { __unstableSetEditorMode, resetZoomLevel } = unlock(
+		useDispatch( blockEditorStore )
+	);
 
 	useEffect( () => {
 		if ( isReady ) {
+			__unstableSetEditorMode( 'edit' );
```

Comment:
Why can't we set an initial state instead? This is triggering a user preference to be set, which in turn triggers a REST API request to store this preference in the database, just by loading the site editor.

Reply:
Ok that's not good. Let's try and find a better fix.

Reply:
If I'm right we no longer want or need to set the editor mode here in `trunk`. If the mode is persisted as a preference then there's no need to reset it here. All we want to do is reset the zoom level.

In WP 6.7 the editor mode is relevant (and is only stored in state) so we can leave "as is".

Reply:
Candidate fix in https://github.com/WordPress/gutenberg/pull/66452.